packages/web/src/pages/explore-page/components/mobile/ExplorePage.tsx

```diff
@@ -238,7 +239,11 @@ const ExplorePage = ({
           isUser
           primaryText={profile.name}
           secondaryText={formatProfileCardSecondaryText(profile.follower_count)}
-          onClick={() => goToRoute(profilePage(profile.handle))}
+          link={link}
+          onClick={(e: React.MouseEvent) => {
```

Comment:
It might make sense for card to call e.preventDefault if link/href provided?

Reply:
I think it already does in the line below, unless you mean e.stopPropogation?